tests/specs/scss/global.scss

```diff
@@ -0,0 +1,15 @@
+@include test-module("Global") {
+   // Testing namespace set functions
+  @include test("config-set [function] Set a single key with namespace.") {
+    $config-namespace: "test-" !global;
+
+    @include config-set("bar", "foo");
+    @include config-set("color", #fff);
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.